src/operators/startWith.ts

```diff
@@ -0,0 +1,7 @@
+import Observable from '../Observable';
+import ScalarObservable from '../observables/ScalarObservable';
+
+export default function startWith<T>(x: T): Observable<T> {
+  const init = ScalarObservable.just(x);
+  return init.concat(null, <Observable<T>>this);
```

Comment:
It'd also be great if you changed this to use the `ScalarObservable` constructor, instead of `just`, since `ScalarObservable` isn't guaranteed to have `just` statically defined either.